chromadb/proto/chroma_pb2.py

```diff
@@ -1,11 +1,22 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: chromadb/proto/chroma.proto
+# Protobuf Python Version: 5.27.2
```

Comment:
i think this version is too new!

Reply:
I've pinned `grpcio` to version 1.58.0 and `protobuf` to 4.21.6. The bindings have been regenerated.